### ISSUE TITLE
download stuck at "checking input"

### ISSUE THREAD

**Issue by ET1982** [hama3254/Crunchyroll-Downloader-v3.0#534]
Hi! I'm using v3.8.9 but today it no longer works
I added some CR's episodes au I usually do, but the download doesn't start, it remains on "Checking input...".
I tried with different series and also with a previous version (v.3.8.8.1) but the result is the same.

I hope CR didn't add some feature on the website that blocks this software :-/

Am I the only one experiencing this issue?

**Comment by ET1982** [comment]
I add some further info.

In the last months I always uses "hybrid mode" because with default mode the software was often crushing. On the contrary, in hybrid mode it almost never crushed again.

I tried today to change mode from hybrid to default but it didn't work anyway. Same thing with Hybrid mode keep cache.

default <-- "the download process seems to have cruched" "Please press play button again to retry"
hybrid <-- stuck at "checking input"
hybrid - keep cache <-- stuck at "checking input"

The rest of settings that always worked fine until the last time are:

error handling 0
resolution 1080p
format mkv (merge soft subs)
CR hardsub null, softsubs italian, default italian

Sorry, I don't know if there's a log and where

**Comment by hama3254** [comment]
please tell me the episode/anime you tried, it would also be helpfull to know if it is the old crunchyroll website or the new beta website with that issue.
further more would the logfile help at best with default download options.

The logfile can be obtained from the right click menu and then either saving it or copying it to the clipboard
![grafik](https://user-images.githubusercontent.com/52826614/136704200-5b10f94f-c74f-408c-9ba9-13d9a3298ddb.png)

**Comment by ET1982** [comment]
Hi hama, thanks for answering.

Thanks to the logfile I understood that I'm an idiot :-)

Basically, a couple of days ago I changed one of my harddrives and I changed also the letter of the download drive (from F: to E:)

Then I updated the folder path on Options/Main/Profile folder, but thanks to the logfile I just saw that on the Add Video window there still was "F:\CR-download" instead of "E:\CR-download" (I didn't noticed because of the E/F similarity...)

So, that was just a false alarm ^^" Sorry for making you wasting time! I take the chance to thank you again for this software, there are others able to download from CR, but this is really the best among them. The 3.8.9 version is much more stable than the previous ones
